lib/gtf/genome.cc

```diff
@@ -73,9 +71,54 @@ int genome::read(const string &file)
 		genes[i].shrink();
 	}
 
+	gzrd.close_gz();
+
 	return 0;
 }
 
+// int genome::read(const string &file)
```

Comment:
That's too much commenting out. Just erase it.

Reply:
I uncommented that function.